Project/VetAppointmentAPI/VetAppointmentAPI/VetAppointment.Infrastructure/Context/DatabaseContext.cs

```diff
@@ -5,10 +5,16 @@ namespace VetAppointment.Infrastructure.Context
 {
     public class DatabaseContext : DbContext, IDatabaseContext
     {
-        //public DatabaseContext()
-        //{
-        //    this.Database.EnsureCreated();
-        //}
+        public DatabaseContext(DbContextOptions<DatabaseContext> options) : base(options)
+        {
+            //this.Database.EnsureDeleted();
+            this.Database.EnsureCreated();
+        }
+        public DatabaseContext()
+        {
```

Comment:
Let's remove all unnecessary comments so it'll be more readable in the long run.